planner-agent/src/workflows/vibeTodoCreator/utils.py

```
@@ -0,0 +1,242 @@
+        # Check if the insertion was successful
+        return result.acknowledged
+    except ConnectionFailure:
+        print("MongoDB connection failed")
+        return False
+    except PyMongoError as e:
+        print(f"MongoDB error: {e}")
+        return False
+
+
+if __name__ == "__main__":
+    task = TaskModel(
```

Ruff (0.11.9)

Undefined name `TaskModel`

(F821)